### ISSUE TITLE
better handling of corrupt files

### ISSUE THREAD

**Issue by Kirk3gaard** [nanoporetech/pod5-file-format#24]
Hi

For some unknown reason we have some fast5 files in a skip folder that appear to be corrupted. 

The pod5 convert runs happily until it reaches one of the "corrupt" files and then crashes completely. I can then manually remove the reported file but have to start over again. 

I would prefer if corrupt files were simply reported and left out of the pod5 conversion.

Best regards
Rasmus

Error nessage exanple:

````
pod5 convert fast5 corrupt_fast5s/ dummy.pod5
Converting 16 fast5 files.. 
0 reads,	 0 Samples,	 0/16 files,	 0.0 MB/s
Error processing corrupt_fast5s/PAK66154_skip_ecff0cbe_52bfab07_177.fast5

Sub-process trace:
Traceback (most recent call last):
  File "/home/ubuntu/.local/lib/python3.10/site-packages/pod5/tools/pod5_convert_from_fast5.py", line 309, in get_reads_from_files
    _f5[read_id],
  File "h5py/_objects.pyx", line 54, in h5py._objects.with_phil.wrapper
  File "h5py/_objects.pyx", line 55, in h5py._objects.with_phil.wrapper
  File "/home/ubuntu/.local/lib/python3.10/site-packages/h5py/_hl/group.py", line 357, in __getitem__
    oid = h5o.open(self.id, self._e(name), lapl=self._lapl)
  File "h5py/_objects.pyx", line 54, in h5py._objects.with_phil.wrapper
  File "h5py/_objects.pyx", line 55, in h5py._objects.with_phil.wrapper
  File "h5py/h5o.pyx", line 190, in h5py.h5o.open
KeyError: "Unable to open object (file read failed: time = Mon Mar 13 15:03:13 2023\n, filename = 'corrupt_fast5s/PAK66154_skip_ecff0cbe_52bfab07_177.fast5', file descriptor = 10, errno = 5, error message = 'Input/output error', buf = 0x557f9c22e9d0, total read size = 80, bytes this sub-read = 80, bytes actually read = 18446744073709551615, offset = 0)"

An unexpected error occurred: 

POD5 has encountered an error: ''

For detailed information set POD5_DEBUG=1'
````

Ended up looping until no more corrupt files were left

````
RUNAGAIN=1;

while [ $RUNAGAIN -gt 0 ]
do
	pod5 convert fast5 ./20230301_1540_1E_PAK66154_ecff0cbe/fast5_skip/*.fast5 out.pod5 2> errout.txt
	corrupt_file=$(cat errout.txt | grep "filename" | sed -E "s/.*filename = '(.*.fast5).*/\1/")
	if  [ -f $corrupt_file ];
	then
	    mv $corrupt_file corrupt_fast5s/;
	    rm errout.txt;
	    rm out.pod5;
	else
	    echo "No more corrupt files"
	    RUNAGAIN=0;
	fi
done
````

**Comment by HalfPhoton** [comment]
Hi Rasmus,

Thanks for reporting this. We'll get this sorted ASAP.

**Comment by HalfPhoton** [comment]
Closing this ticket as it should be resolved in `0.1.13` and `0.1.14`
